ElasticSearch/legos/elasticsearch_get_index_health/elasticsearch_get_index_health.py

```diff
@@ -51,59 +51,35 @@ def elasticsearch_get_index_health(handle, index_name="") -> Tuple:
     :return: A list of dictionaries where each dictionary contains stats about each index
     """
     try:
-        indices_output = []
-        # If no specific index is provided, get all indices
-        if len(index_name)==0 :
-            indices_output = handle.web_request("/_cat/indices?h=index", "GET", None)
-            indices_output = ''.join(indices_output).split('\n')
-        # If a specific index is provided, only consider that index
+        if index_name:
+            index_names = [index_name]
         else:
-            indices_output.append(index_name)
+            # Fetches all indices; ensure only non-empty lines and non-system indices are considered
+            response = handle.web_request("/_cat/indices?h=index", "GET", None)
```

Comment:
curl -X GET "http://lightbeam-elasticsearch-master.lightbeam.svc.cluster.local:9200/_cat/indices?v&h=index,status"
Use this